errors.go

```diff
@@ -17,6 +17,7 @@ const (
 	errUnmarshalErrorBody                     = "error unmarshalling the JSON response error body"
 	errRequestNotSuccessful                   = "error reported by API"
 	errMissingAccountID                       = "required missing account ID"
+	errMissingMemberRolesOrPolicies           = "account member must be created with roles or policies (not both)"
```

Comment:
these are fine to put in the `account_members` files too if they are specific to just that resource.